src/index.html

```diff
@@ -3,11 +3,333 @@
+          </p>
+        </article>
+
+        <article class="recommended">
+          <img
+            src="./images/products/product_3.png"
+            alt="SoundLink Color Bluetooth speaker II"
+            class="recommended__image"
+          >
+
+          <h3 class="title title--tertiary recommended__title">
+            SoundLink Color Bluetooth speaker II
+          </h3>
+
+          <p class="paragraph recommended__paragraph">
+            Portable bluetooth
+            <br>
+            <span class="paragraph paragraph--bold">$ 129.00</span>
+          </p>
+        </article>
+      </section>
+
+      <h2 class="title title--secondary main__title" id="categories">Browse Bose products by category</h2>
+
+      <section class="section main__section" >
+        <article class="category">
+          <div class="category__square category__square--big">
+            <img
+              src="./images/categories/category_1.jpg"
+              alt="Smiling woman with headphones"
+              class="category__image"
+            >
+          </div>
+
+          <div class="category__square category__square--small">
+            <img
+              src="./images/categories/category_2.jpg"
+              alt="Busy woman in headphones"
+              class="category__image"
+            >
+          </div>
+
+          <h3 class="title title--tertiary category__title">Headphones & earbuds</h3>
+        </article>
+
+        <article class="category">
+          <div class="category__square category__square--big">
+            <img
+              src="./images/categories/category_3.jpg"
+              alt="Small portable bluetooth speaker"
+              class="category__image"
+            >
+          </div>
+
+          <div class="category__square category__square--small">
+            <img
+              src="./images/categories/category_4.jpg"
+              alt="Big portable bluetooth speaker"
+              class="category__image"
+            >
+          </div>
+
+          <h3 class="title title--tertiary category__title">Speakers</h3>
+        </article>
+
+        <article class="category">
+          <div class="category__square category__square--small">
+            <img
+              src="./images/categories/category_5.jpg"
+              alt="Man in audio sunglasses"
+              class="category__image"
+            >
+          </div>
+
+          <div class="category__square category__square--big">
+            <img
+              src="./images/categories/category_6.jpg"
+              alt="Woman in audio sunglasses"
+              class="category__image"
+            >
+          </div>
+
+          <h3 class="title title--tertiary category__title">Audio sunglasses</h3>
+
+          <button class="button category__button">View all</button>
+        </article>
+      </section>
+
+      <section class="section main__section" id="how-to-buy">
+        <article class="buy-info">
```

Comment:
Would consider changing it to just some div rather than article, judge it youself [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)